src/util/settings/theme.cc

```diff
@@ -88,28 +89,14 @@ QColor chunkBackground(int color_index) {
   }
   QColor ret =
       chunkBackgroundColors_[color_index % chunkBackgroundColors_.size()];
-  if (isDark()) {
-    ret = QColor(ret.red() ^ 0xff, ret.green() ^ 0xff, ret.blue() ^ 0xff);
-  }
 
-  return ret;
+  return colorInvertedIfDark(ret);
 }
 
 QColor byteColor(uint8_t byte) {
-  if (isDark()) {
-    byte ^= 0xFF;
-  }
-
-  int red, blue, green = 0;
-  if (byte <= 0x80) {
-    blue = 0xff;
-    red = qMin(0xff, byte * 2);
-  } else {
-    blue = qMin(0xff, (0xff - byte) * 2);
-    red = 0xff;
-  }
-
-  return colorInvertedIfDark(QColor(red, green, blue));
+  QColor color = util::settings::hexedit::colorOfText();
+  color.setAlpha(byte / 2 + 127);
```

Comment:
Please don't use `setAlpha` - we don't want to blend it with the color of the background (which is not always dark/white). IMO we should use a constant color gradient (independent of the background), the current version is hard to read when there are some chunks added (they change byte background).

Just decide on the gradient parameters and calculate the color ;) Doing this should be quite easy using getHsv/setHsv  methods (or getHsl/setHsl). Or just use QColor::lighter / QColor::darker which does the same.

Reply:
I'm still working on this.

Reply:
Ok, I'll wait for the rest and then do next review iteration

Reply:
I changed `setAlpha` to `QColor::darker` it's allow choose color with max brightness.